src/pages/index.js

```diff
@@ -89,13 +142,32 @@ profileAddButton.addEventListener("click", () => {
 });
 
 function handleAddFormSubmit(data) {
+  addPopupForm.setLoading(true, "Saving...");
   const name = data.name;
   const link = data.link;
-  const card = generateCard({ name, link });
 
-  cardListSection.addItem(card);
-  addForm.reset();
-  addPopupForm.close();
+  api
+    .addCard({ name, link })
+    .then((newCard) => {
+      const card = generateCard(newCard);
+      cardListSection.addItem(card);
+      addForm.reset();
+      addPopupForm.close();
+    })
+    .catch(console.error)
+    .finally(() => {
+      addPopupForm.setLoading(false, "Saving...");
+    });
+}
+
+function deleteCurrentCard(card) {
+  api
+    .deleteCard(card._cardData._id)
```

Comment:
`_cardData`

Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.


Please, correct it in `Card` as well

Reply:
Another remark hasn't been corrected